web/build.gradle

```diff
@@ -46,6 +46,7 @@ dependencies {
     testCompile group: 'org.apache.logging.log4j', name: 'log4j-api', version: '2.1'
     testCompile group: 'org.json', name:'json', version:'20090211'
     testCompile group: 'com.microsoft.azure', name: 'azure-storage', version: '2.1.0'
+    testCompile group: 'com.google.guava', name: 'guava', version: '20.0'
```

Comment:
What's this?

Reply:
I added this because I could not run individual tests without this